docs/feature-gates.md

```diff
@@ -30,39 +30,40 @@ edit the Agent configuration in the
 
 ## List of Available Features
 
-| Feature Name                  | Component          | Default | Stage | Alpha Release | Beta Release | GA Release | Extra Requirements | Notes                                         |
-| ----------------------------- | ------------------ | ------- | ----- | ------------- | ------------ | ---------- | ------------------ | --------------------------------------------- |
-| `AntreaProxy`                 | Agent              | `true`  | GA    | v0.8          | v0.11        | v1.14      | Yes                | Must be enabled for Windows.                  |
-| `EndpointSlice`               | Agent              | `true`  | GA    | v0.13.0       | v1.11        | v1.14      | Yes                |                                               |
```

Comment:
I think that one thing that is worth pointing out is that even though we are keeping this FeatureGate around, setting it to `False` will have no effect any more (unlike the `AntreaProxy` FeatureGate for example). So I think the current warning is not sufficient for `EndpointSlice`. If the FeatureGate is explicitly set to `False` in the config, we should log an error stating that the configuration will be ignored.

Reply:
Can you add "Cannot be disabled" in the Notes (last column of this table)?

Reply:
We should still mention it, as otherwise there is no way for a user to tell simply by looking at the documentation

Reply:
ping for this

Reply:
Added  `Locked to `true` (cannot be disabled).` to column ` Notes`.